legal-api/src/legal_api/reports/business_document.py

```diff
@@ -588,25 +593,40 @@ def _get_legal_type_description(legal_type: str) -> str:
                 'BEN': 'Voluntary Dissolution',
                 'ULC': 'Voluntary Dissolution',
                 'CC': 'Voluntary Dissolution',
+                'C': 'Voluntary Dissolution',
+                'CBEN': 'Voluntary Dissolution',
+                'CUL': 'Voluntary Dissolution',
+                'CCC': 'Voluntary Dissolution',
```

Comment:
I added all C types as was not sure if to let it fall back on default name. I can remove it if you think that is the way to go